### PR TITLE
Improve metainfo

### PR DESCRIPTION
**Pull request by Alexander-Wilms**
I added some more tags according to the Freedesktop specification: https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html

Perhaps the screenshots should be replaced as well, since they use various themes, some cropped, one without window decoration and one with and don't have a 16:9 ratio.

I found these models shown in the current screenshots:
* https://github.com/bq/prusa-i3-hephestos/blob/master/freecad/prusa-hephestos-assembly.fcstd
* https://forum.freecad.org/viewtopic.php?p=377075#p377075 (sadly without a link to the actual model)
* https://wiki.freecad.org/Arch_tutorial#Conclusion
* https://wiki.freecad.org/Manual:Modeling_for_product_design

Perhaps the examples could be added here as well (https://github.com/FreeCAD/Examples) and then be reference in XML comments in the metainfo file.

### CODE REVIEW

File: src/XDGData/org.freecad.FreeCAD.metainfo.xml.in [hunk 1]
```diff
@@ -40,22 +40,43 @@
...
+  <keywords>
+    <!-- copied from https://www.freecad.org/ HTML: <meta name="keywords" [...] />  -->
+    <keyword>engineering</keyword>
+    <keyword translate="no">opencascade</keyword>
+    <keyword >architecture</keyword>
+    <keyword translate="no">cad</keyword>
+    <keyword translate="no">bim</keyword>
+    <keyword translate="no">fem</keyword>
+    <keyword translate="no">3d</keyword>
+    <keyword translate="no">3d-printing</keyword>
```
Comment (adrianinsaval) [inline src/XDGData/org.freecad.FreeCAD.metainfo.xml.in:68]: why no translation of this?

**Comment by luzpaz** [comment]
Screenshot_20230914_074548

Does this change break dwnstream package repos that are relying on finding an `*appdata.xml` and not a `*metainfo.xml` ?

**Comment by adrianinsaval** [comment]
Is there any benefit to moving to metainfo.xml? If not it might be better to just leave it as is since appdata.xml is also supported

**Comment by adrianinsaval** [comment]
please update this too in this PR: https://github.com/FreeCAD/FreeCAD/blob/10b3ae382ff46103f89ea92722040bad20612373/package/fedora/freecad.spec#L323-L324

**Comment by Alexander-Wilms** [comment]
Done

**Pull request merged**